tests/integration/test_keeper_map/test.py

```diff
@@ -47,13 +47,25 @@ def remove_children(client, path):
 
 
 def test_create_keeper_map(started_cluster):
+    node.query("DROP TABLE IF EXISTS test_keeper_map SYNC")
```

Comment:
that's already being done, this was to prevent the failure of subsequent tests after this one fails.

> All other cases are strange and we need to fix it anyway.

but I see your point and agree

Reply:
You can just rename table to fix. Order of tests may change, so we don't rely on dependencies of tests.